Formula/bigloo.rb

```diff
@@ -21,19 +20,30 @@ class Bigloo < Formula
   depends_on "autoconf" => :build
   depends_on "automake" => :build
   depends_on "libtool" => :build
+  depends_on "pkg-config" => :build
 
+  depends_on "bdw-gc"
   depends_on "gmp"
+  depends_on "libunistring"
+  depends_on "libuv"
+  depends_on "openjdk"
   depends_on "openssl@1.1"
+  depends_on "pcre"
 
   def install
+    inreplace "configure" do |s|
+      s.gsub!(/^\s+custom\w+=(yes)$/, "no")
```

Comment:
```suggestion
      s.gsub!(/^(\s+custom\w+)=yes$/, "\\1=no")
```

Reply:
Can be simplified to one line too problably: `inreplace "configure", /^(\s+custom\w+)=yes$/, "\\1=no"`